ci-operator/config/openshift/osde2e/openshift-osde2e-main.yaml

```
@@ -403,6 +403,30 @@ tests:
     name: osde2e-common
   - mount_path: /usr/local/osde2e-credentials
     name: osde2e-credentials
+- as: rosa-stage-complete-e2e
+  commands: |
+    export REPORT_DIR="$ARTIFACT_DIR"
+    export CONFIGS="rosa,stage,complete-e2e"
+    export CHANNEL="stable"
+    export SECRET_LOCATIONS="/usr/local/osde2e-common,/usr/local/osde2e-credentials,/usr/local/osde2e-rosa-stage"
+    export HARNESS_TIMEOUT="900"
+    export POLLING_TIMEOUT="7200"
+    export ROSA_ENV="stage"
+    export ROSA_STS="true"
+    export ROSA_AWS_REGION="random"
```

```suggestion
    export REPORT_DIR="$ARTIFACT_DIR"
    export CONFIGS="rosa,sts,stage,complete-e2e"
    export CHANNEL="stable"
    export SECRET_LOCATIONS="/usr/local/osde2e-common,/usr/local/osde2e-credentials,/usr/local/osde2e-rosa-stage"
    export HARNESS_TIMEOUT="900"
    export POLLING_TIMEOUT="7200"
    export ROSA_AWS_REGION="random"
```

We can use the new [sts](https://github.com/openshift/osde2e/blob/main/configs/sts.yaml) config from osde2e and the [stage](https://github.com/openshift/osde2e/blob/main/configs/stage.yaml#L3-L4) now also contains the setting for telling ROSA to use stage over needing to set the `ROSA_ENV` variable.